client/gutenberg/editor/index.js

```diff
@@ -18,11 +18,13 @@ export default function () {
 	page( '/post/new', () => page.redirect( '/post' ) ); // redirect from beep-beep-boop
 	page( '/post/:site/:post?', siteSelection, authenticate, post, makeLayout, clientRender );
 	page( '/post/:site?', siteSelection, makeLayout, clientRender );
+	page.exit( '/post/:site?/:post?', exitPost );
```

Comment:
I'm adding this middleware because it was in the previous post-editor component. I'm no getting totally why we need it, so looking into worths it.

Reply:
I did a bit of research into this, and it essentially clears the `postId` from the state. There is a `startEditingPost` action that occurs in the calypsoify iframe, which does the opposite, and sets the `postId` in the state. This is apparently for components like editor revisions, but I assume that these work outside of Gutenframe, probably via the editing toolkit.

Even if that's not the case, it's not being cleared currently when people leave the Gutenframe editor, so it's probably not essential, but I suppose it's good to keep things tidy.

I'll do a bit more investigation into the editor components using it, so we can make a final decision.